### DISCUSSION TITLE
What's the wanted array comma style?

### DISCUSSION THREAD

**Discussion by PHPGuus** [spatie/laravel-ignition#97] [Q&A] [answered]
Hey there,

just filed a PR for some minor change, and saw that some arrays have a comma after the last element, and some arrays don't. What's the preferred coding style for this? I'll do a PR to pull everything into that preferred coding style.

Best regards,
Guus

**Comment by freekmurze** [comment, marked as answer]
We prefer having comma's after the last element. Don't worry about it, I'll check if we can automatically fix this (did it manually now for your PR)

**Comment by PHPGuus** [comment]
That's what I thought, but wasn't entirely sure. Thanks, Freek.
